### PR TITLE
[MODORDERS-943] - Encumbrance status becomes "Unreleased" after changing fund distribution

### PR DESCRIPTION
**Pull request by Abdulkhakimov**
## Purpose
_Describe the purpose of the pull request._

## Approach
_How does this change fulfill the purpose?_

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Comment by Abdulkhakimov** [comment]
image

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-encumbrance-status-after-moving-expended-value.feature [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    Then status 200
+    And match $.amount == <amount>
+    And match $.encumbrance.amountExpended == <amountExpended>
+    And match $.encumbrance.status == <status>
+
+    Examples:
+      | index | amount | amountExpended | status       |
+      | 0     | 0      | 0.5            | 'Released'   |
+      | 1     | 0      | 0.5            | 'Released'   |
+      | 2     | 0.4    | 0              | 'Unreleased' |
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/mod-orders/features/check-encumbrance-status-after-moving-expended-value.feature:189]: please add some comment or log etc to describe why these values are expected values

Comment (Abdulkhakimov) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/check-encumbrance-status-after-moving-expended-value.feature:200]: commented

**Review by Dmitriy-Butramyou — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/mod-orders/features/check-encumbrance-status-after-moving-expended-value.feature (hunk 1 above)
Comment (SerhiiNosko) [reply acquisitions/src/main/resources/thunderjet/mod-orders/features/check-encumbrance-status-after-moving-expended-value.feature:200]: thank you, not it is easier to understand

**Pull request merged**